index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <nav class="toolbar">
+    <!-- <div class="logo"><a href="">Tsige Wossne</a></div> -->
+    <div class="menu_main">
+      <input type="checkbox" class="toggler">
+      <div class="humbuger"></div>
+      <div class="menu">
+        <div class="overlay">
+          <div>
+            <ul>
+              <li><a href="">Home</a></li>
+              <li><a href="">Projects</a></li>
+              <li><a href="">Contribution</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </nav>
```

Comment:
1. [optional] Please remove unnecessary comments(line 17) from your code to have a clean code free from errors and confusion 😄 
2. As suggested by the previous reviewer, you should follow the right structure for your code . You should use the `header` for the outer section for your `toolbar`, followed by your logo in a `<a href="#"></a>` tag , then your `nav` tag with your `hamburger menu` in it . Like this:
    >  <body>
         <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
           </header>
          <section>
               --- the headline section (right after the header) ----
           </section>
      </body>